realm/realm-library/src/main/cpp/java_network_transport.hpp

```diff
@@ -92,7 +103,7 @@ struct JavaNetworkTransport : public app::GenericNetworkTransport {
 
             jint http_code = env->CallIntMethod(response, get_http_code_method);
             jint custom_code = env->CallIntMethod(response, get_custom_code_method);
-            JStringAccessor java_body(env, (jstring) env->CallObjectMethod(response, get_body_method));
+            JStringAccessor java_body(env, (jstring) env->CallObjectMethod(response, get_body_method), true);
```

Comment:
Any reason to introduce the `owned` boolean for just this case? Couldn't we delete it after calling the ref after calling the `completionBlock`?

Reply:
Just taking advantage of the new constructor needed to delete all the temporary `jstring` references generated in the `JObjectArrayAccessor`. We could do it manually, but I assume that we have loads of `JStringAccessor`s that received a `jstring` which could instantly be deleted, so I see no reason for removing nice functionality. Alternative is to define a local `jstring` and manually delete the reference afterwards.